models/cryptopunks/ethereum/cryptopunks_ethereum_punk_transfers.sql

```diff
@@ -0,0 +1,10048 @@
+{{ config(
+        alias ='current_owners_per_token',
+        partition_by = ['evt_block_time'],
```

Comment:
i wouldn't partition by a timestamp, that is much too granular. consider creating a new column for day/week/month, then partition by that. it depends on data volume per each time segment. if not many tx's happen, i'd consider monthly. if it's heavy volume, maybe daily

Reply:
Got it just updated! Thank you